revolution/display.py

```diff
@@ -1,15 +1,379 @@
-from dataclasses import dataclass
+from dataclasses import dataclass, field
 from logging import getLogger
 from typing import ClassVar
+from time import sleep
 
 from revolution.application import Application
 from revolution.environment import Endpoint
 
+import st7565  # type:ignore
+import xglcd_font as font  # type:ignore
+
 _logger = getLogger(__name__)
 
 
 @dataclass
 class Display(Application):
     endpoint: ClassVar[Endpoint] = Endpoint.DISPLAY
+    d1: st7565.Glcd = field(init=False)
+    d2: st7565.Glcd = field(init=False)
+    JetBrains13x20: font.XglcdFont = field(init=False)
+    JetBrains13x21: font.XglcdFont = field(init=False)
+    Font5x7: font.XglcdFont = field(init=False)
+    Liberation_Sans20x28: font.XglcdFont = field(init=False)
+    current_cruise: bool
+
+    # display design based on google slides that you can find here
+    # https://docs.google.com/presentation/d/1P_kAPFkVbpgUW_R5nXAC8cFNGyiidCfmwuXg6XJ8BWc/edit?usp=sharing
+
+    # instantiate the two object for two display and run init for controller
+    # also create Font object to later write with
+    def __post_init__(self) -> None:
+        self.d1 = st7565.Glcd()
+        self.d2 = st7565.Glcd()
+        self.d1.init()
+        self.d2.init()
+        # TODO generate new JetBrains13x21 font to have all characters
+        self.JetBrains13x20 = font.XglcdFont("display_data/JetBrains_Mono13x20.c", 13, 20, 97)  # noqa: E501
+        self.JetBrains13x21 = font.XglcdFont("display_data/JetBrains_Mono13x21_Symbol.c", 13, 21, 32)  # noqa: E501
+        self.Liberation_Sans20x28 = font.XglcdFont("revolution/display_data/Liberation_Sans20x28_Numbers.c", 20, 28, 46)  # noqa: E501
+        self.Font5x7 = font.XglcdFont("display_data/font5x7.c", 5, 7, 32)
+
+    # draw blue sky logo, wait for a while, then draw the default screen
+    def startup(self) -> None:
```

Comment:
Should be run as a separate thread. This was done in def setup in other ``Application``s.